services/QuillLMS/engines/evidence/spec/lib/prefilter_check_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+module Evidence
+  RSpec.describe(PrefilterCheck) do
+    let(:rule_factory_overrides) { {rule_type: 'prefilter', universal: true, optimal: false} }
+
+    describe '#initialize' do
+      it 'should retrieve associated Rules' do
+        rule = create(:evidence_rule, rule_type: 'prefilter')
+        prefilter_check = Evidence::PrefilterCheck.new("entry")
+        expect(prefilter_check.prefilter_rules.first).to eq(rule)
+      end
+    end
+
+    describe '#feedback_object' do
+      let!(:prefilter_rule1) do 
+        create(:evidence_rule, **rule_factory_overrides, uid: '123')
+      end
+
+      let(:non_prefilter_rule) { create(:evidence_rule, **rule_factory_overrides, rule_type: 'notPrefilter') }
+
+      context 'PREFILTERS query hit' do 
+        before do 
+          Evidence::PrefilterCheck && stub_const('Evidence::PrefilterCheck::PREFILTERS', {
+            '123' => ->(x) { false }
```

Comment:
i prefer using the keyword `lambda`, but Rubocop wants the arrow syntax for one-liners